### PR TITLE
Update custom.js

### PR DESCRIPTION
**Pull request by mananpal1997**
Fix for Issue [#27](https://github.com/sahildua2305/hackIDE/issues/27)

### CODE REVIEW

File: hackIDE/static/hackIDE/js/custom.js [hunk 1]
```diff
@@ -113,13 +113,8 @@ $(document).ready(function(){
...
 		zip.file(filename+"."+ext, text)
-		var xhr = new XMLHttpRequest();
-		xhr.onload = function() {
-			var downloaded = zip.generate({type : "blob"})
-			saveAs(downloaded, "test.zip")
-		}
-		xhr.open('get', 'https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-alpha1/jquery.min.js')
-		xhr.send()
+		var downloaded = zip.generate({type : "blob"})
+		saveAs(downloaded, "test.zip")
```
Comment (sahildua2305) [inline hackIDE/static/hackIDE/js/custom.js]: @mananpal1997 Where is `saveAs` function defined?

File: hackIDE/static/hackIDE/js/custom.js [hunk 2]
```diff
@@ -536,6 +531,17 @@ $(document).ready(function(){
...
+	var mouse_inside = false;
+	$('#settings-pane').hover(function(){
+		mouse_inside = true;
+	}, function(){
+		mouse_inside = false;
+	});
+	$('body').mouseup(function(){
+		if(!mouse_inside)
+			$('#settings-pane').hide();
+	});
```
Comment (sahildua2305) [inline hackIDE/static/hackIDE/js/custom.js]: @mananpal1997 Have you properly tested these changes on local?

**Comment by sahildua2305** [comment]
@mananpal1997 Thanks!

Please check my comments on the code.

File: hackIDE/static/hackIDE/js/custom.js (hunk 2 above)
Comment (mananpal1997) [inline hackIDE/static/hackIDE/js/custom.js]: Yes

File: hackIDE/static/hackIDE/js/custom.js (hunk 1 above)
Comment (mananpal1997) [inline hackIDE/static/hackIDE/js/custom.js]: saveAs function is in FileSaver.js

Comment (sahildua2305) [inline hackIDE/static/hackIDE/js/custom.js]: Have you made any change in `FileSaver.js` or was it already there?

Comment (mananpal1997) [inline hackIDE/static/hackIDE/js/custom.js]: No changes there !

**Pull request merged**